src/app/screens/Accounts/Detail/index.tsx

```diff
@@ -481,6 +503,38 @@ function AccountDetail() {
                 />
               </div>
             </div>
+            <MenuDivider />
+            <div
+              className={`flex justify-between items-end ${
+                !hasMnemonic && "opacity-30"
+              }`}
+            >
+              <div className="w-7/12">
+                <p className="text-gray-900 dark:text-white font-medium">
+                  {t("liquid.enable.title")}
+                </p>
+                <p className="text-gray-500 text-sm dark:text-neutral-500">
+                  {t("liquid.enable.description")}
+                </p>
+              </div>
+
+              <div className="w-1/5 flex justify-end align-middle">
+                <Toggle
```

Comment:
@louisinger what is the reason you added the toggle? we are thinking maybe it is more user-friendly not to have it. Then anyone who sets up a master key can use liquid immediately. CC @reneaaron @stackingsaunter

Reply:
:+1: +1 to remove the toggle step ! I just did it to respect what u did with the `window.liquid` provider (to do not inject by default into webpages)

Reply:
We reviewed and it is only a small script being injected. We don't think it is an issue

Reply:
I've removed the toggle